bitnami/nginx/templates/_helpers.tpl

```diff
@@ -112,3 +113,25 @@ nginx: missing-extra-volume-mounts
     the extraVolumeMounts value
 {{- end -}}
 {{- end -}}
+
+{{/*
+Validate values of NGINX - If PSP is enabled RBAC should be enabled too
+*/}}
+{{- define "nginx.validateValues.psp" -}}
+{{- if and .Values.psp.create (not .Values.rbac.create) }}
+nginx: psp.create, rbac.create
+    RBAC should be enabled if PSP is enabled in order for PSP to work.
+    More info at https://kubernetes.io/docs/concepts/policy/pod-security-policy/#authorizing-policies
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the appropriate apiVersion for podsecuritypolicy.
+*/}}
+{{- define "podsecuritypolicy.apiVersion" -}}
+{{- if semverCompare "<1.10-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "extensions/v1beta1" -}}
+{{- else -}}
+{{- print "policy/v1beta1" -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Could we add this to the `common` chart?

https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_capabilities.tpl

Reply:
@dani8art done